examples/apis.json

```diff
@@ -22,10 +22,6 @@
     "commented": true,
     "importName": "GenericComponent"
   },
-  "camera": {
-    "func": "get_image",
-    "packagePath": "viam.components"
-  },
```

Comment:
do not remove this, but replace with `get_images`

Reply:
got it. Addressed